### PR TITLE
Add release notes for v0.12.1

### PR DESCRIPTION
**Pull request by gep13**

### CODE REVIEW

**Review by JPRuskin — APPROVED** [review]
LGTM!

File: input/en-us/choco/release-notes.md [hunk 1]
```diff
@@ -12,6 +12,12 @@ This covers changes for the "chocolatey" and "chocolatey.lib" packages, which ar
 
 > :memo: **NOTE** For commercial editions, please also refer to [Licensed Release Notes](xref:setup-licensed).
 
+## [0.12.1](https://github.com/chocolatey/choco/issues?q=is%3Aclosed+milestone%3A0.12.1) (January 25th, 2022)
+### BUG FIXES
+ * Fix - Installation of Chocolatey fails when running under "RemoteSigned" execution policy - see [#2539](https://github.com/chocolatey/choco/issues/2539)
```
Comment (AdmiringWorm) [inline input/en-us/choco/release-notes.md:17]: Wasn't this for `AllSigned` only, or did something change?

File: input/en-us/choco/release-notes.md [hunk 2]
```diff
@@ -12,6 +12,12 @@ This covers changes for the "chocolatey" and "chocolatey.lib" packages, which ar
...
+ * Fix - Output from "choco template -h" references wrong subcommand in usage statement - see [#2546](https://github.com/chocolatey/choco/issues/2546)
```
Comment (AdmiringWorm) [inline input/en-us/choco/release-notes.md:18]: 👍

File: input/en-us/choco/release-notes.md (hunk 1 above)
Comment (gep13) [reply input/en-us/choco/release-notes.md]: I have updated this.

**Review by AdmiringWorm — APPROVED** [review]
LGTM 👍

**Pull request merged**